application/dash_application/utility/sidebar_utils.py

```diff
@@ -12,7 +12,7 @@
 import gsiqcetl.column
 from . import df_manipulation as df_tools
 
-total_reads_cutoff_label = "Passed Filter Reads (* 10^6) minimum"
+total_clusters_cutoff_label = "Passed Filter Clusters (* 10^6) minimum"
```

Comment:
Is this different from clusters_per_sample_cutoff_label at line 19? 'Clusters Per Sample' was the wording requested in the requirements document.

Reply:
Fixed.